src/main/java/entities/Course.java

```diff
@@ -24,12 +26,14 @@ public Course(
             ArrayList<Section> lectures,
             ArrayList<Section> tutorials,
             char session,
-            ArrayList<String> exclusions) {
+            ArrayList<String> exclusions,
```

Comment:
We should stick to Liskov substitution principle and code to interfaces. I'd change these to List

Reply:
Yes Sir! :D